sdcm/sct_events/monitors.py

```diff
@@ -49,9 +51,10 @@ def __init__(self, raw_alert: dict, severity: Severity = Severity.WARNING):
         self.alert_name = self.labels.get("alertname", "")
 
         try:
-            self.severity = Severity[self.labels["sct_severity"]]
-        except KeyError:
-            pass
+            if "sct_severity" in self.labels:
+                self.severity = Severity[self.labels["sct_severity"]]
```

Comment:
I think walrus operator would work nicely here, and there would be no need for the try/except clause.
```python
            if sct_severity:= self.labels.get("sct_severity"):
                self.severity = Severity[sct_severity]
```